added docu for second prometheus instance (#9837)
kyma-project/kyma: docs/monitoring/02-01-monitoring.md

2. [**Prometheus**](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. To obtain such metrics, Prometheus uses the [**kube-state-metrics**](https://github.com/kubernetes/kube-state-metrics) service. It generates the metrics from Kubernetes API objects and exposes them on the `/metrics` HTTP endpoint.  
Pods can also contain applications with custom metrics, such as the total storage space available in the MinIO server. Prometheus stores this polled data in a time-series database (TSDB) and runs rules over them to generate alerts if it detects any metric anomalies. It also scrapes metrics provided by [**Node Exporter**](https://github.com/mindprince/nvidia_gpu_prometheus_exporter) which exposes existing hardware metrics from external systems as Prometheus metrics.

>    >**NOTE:** Besides this main Prometheus instance, there is a second Prometheus instance running in the 'kyma-system' namespace responsible for collecting and aggregating [Istio service-mesh metrics](#istio-monitoring).

Reviewer: ```suggestion
   >**NOTE:** Besides this main Prometheus instance, there is a second Prometheus instance running in the 'kyma-system' namespace responsible for collecting and aggregating [Istio service-mesh metrics](#details-istio-monitoring).
```

>    >**NOTE:** Besides this main Prometheus instance, there is a second Prometheus instance running in the 'kyma-system' namespace responsible for collecting and aggregating [Istio service-mesh metrics](#istio-monitoring).

Author: done

>    >**NOTE:** Besides this main Prometheus instance, there is a second Prometheus instance running in the 'kyma-system' namespace responsible for collecting and aggregating [Istio service-mesh metrics](#details-istio-monitoring).

Author: ```suggestion
   >**NOTE:** Besides this main Prometheus instance, there is a second Prometheus instance running in the 'kyma-system' namespace responsible for collecting and aggregating [Istio Service Mesh metrics](#details-istio-monitoring).
```

>    >**NOTE:** Besides this main Prometheus instance, there is a second Prometheus instance running in the 'kyma-system' namespace responsible for collecting and aggregating [Istio Service Mesh metrics](#details-istio-monitoring).

Author: ```suggestion
   >**NOTE:** Besides this main Prometheus instance, there is a second Prometheus instance running in the `kyma-system` Namespace. This second instance is responsible for collecting and aggregating [Istio Service Mesh metrics](#details-istio-monitoring).
```
